purchase/app/purchase/CheckoutAddressFormData.java

```diff
@@ -0,0 +1,233 @@
+package purchase;
+
+import io.sphere.sdk.models.Base;
+import play.data.validation.Constraints;
+
+public class CheckoutAddressFormData extends Base {
+    public interface Validation {}
+
+    private String csrfToken;
```

Comment:
I thought we moved the token to meta: https://github.com/sphereio/sphere-sunrise-design/blob/master/input/templates/partials/common/meta.json